src/services/pallets/PalletsStakingProgressService.ts

```diff
@@ -100,7 +112,9 @@ export class PalletsStakingProgressService extends AbstractService {
 			...baseResponse,
 			nextActiveEraEstimate: nextActiveEra.toString(10),
 			electionStatus: {
```

Comment:
I think it is more clear to have `electionStatus: "Deprecated, see docs"` instead of having the inner `status` property say that. RN it jus looks like the inner `status` field is deprecated and not the whole `electionStatus`

Reply:
Awesome, fixed it up so that electionStatus return the message instead.